dev/EVAndPATH/Microsoft.Process.Environment/Microsoft.Process.Environment.def

```diff
@@ -0,0 +1,3 @@
+EXPORTS
+DllCanUnloadNow = WINRT_CanUnloadNow                    PRIVATE
+DllGetActivationFactory = WINRT_GetActivationFactory    PRIVATE
```

Comment:
NIT: Github's review/diff shows a little red circle-with-line-threw-it on the last line because "No newline at end of file"

Reply:
Added newline.